LayoutTests/platform/glib/accessibility/radio-button-group-dynamic-changes-expected.txt

```diff
@@ -0,0 +1,7 @@
+CONSOLE MESSAGE: TypeError: null is not an object (evaluating 'radio2a.linkedUIElementAtIndex(0).isEqual')
+FAIL Unexpected error: TypeError: null is not an object (evaluating 'radio2a.linkedUIElementAtIndex(0).isEqual')
+FAIL successfullyParsed should be true. Was false.
+Some tests failed.
+
+TEST COMPLETE
```

Comment:
It's typically best not to check in failures for non-WPTs. Linked UI elements is mostly a macOS / iOS concept I believe, so we should just skip this test for glib platforms.